### PR TITLE
css changes in North, South and Notification modals and hover effect for bootstrap icons

### PR DESCRIPTION
**Pull request by monika-Dianomic**
**Pending:**

- [x] Enable *step3 alignment fix in North/South/Notification instance wizard, (also check North service vs  interval based task)
- [x] Fix bi-plus-circle blurry icon

### CODE REVIEW

File: src/app/components/core/notifications/notification-modal/notification-modal.component.html [hunk 1]
```diff
@@ -3,8 +3,8 @@
   <div class="modal-card">
     <header class="modal-card-head">
       <span *ngIf="notification" class="modal-card-title title-name">{{notification.name}}
-        <span><i class="bi bi-info-circle plugin-info" aria-hidden="true" [title]="'Rule Plugin: ' + notification.rule + '&#013;Action Plugin: ' + notification.channel"></i></span>
-        <p class="help has-text-grey ctrl-pipeline">Notification Instance</p>
```
Comment (praveen-garg) [inline src/app/components/core/notifications/notification-modal/notification-modal.component.html:7]: This ctrl-pipeline class css can be removed, or used somewhere?!

**Review by praveen-garg — COMMENTED** [review]
Also fix Enable label alignment in wizard

File: src/app/components/core/notifications/notification-modal/notification-modal.component.html (hunk 1 above)
Comment (monika-Dianomic) [reply src/app/components/core/notifications/notification-modal/notification-modal.component.html:7]: It is being used in `add-pipeline-filter`. I've removed its unwanted occurrences.

File: src/app/components/core/north/north-task-modal/north-task-modal.component.css [hunk 2]
```diff
@@ -100,3 +100,13 @@ a {
   max-width: 1344px;
   margin: auto;
 }
+
+.modal-card-head {
+  padding: 15px !important;
```
Comment (praveen-garg) [inline src/app/components/core/north/north-task-modal/north-task-modal.component.css:105]: how about 12 px?

File: src/app/components/core/notifications/notification-modal/notification-modal.component.html (hunk 1 above)
Comment (praveen-garg) [reply src/app/components/core/notifications/notification-modal/notification-modal.component.html:7]: OK

File: src/app/components/core/north/add-task-wizard/add-task-wizard.component.html [hunk 3]
```diff
@@ -149,7 +149,7 @@
           <div class="column">
             <div class="config-div">
               <div class="field is-horizontal">
-                <div class="field-label has-text-left">
+                <div class="field-label has-text-centered enabled-field">
```
Comment (praveen-garg) [inline src/app/components/core/north/add-task-wizard/add-task-wizard.component.html:152]: step1 labels are  right,
step2 labels are  left,
step2 labels are  centered?!,

Comment (monika-Dianomic) [reply src/app/components/core/north/add-task-wizard/add-task-wizard.component.html:152]: Yes, `has-text-centered` fixed the alignment issue in step3.

File: src/styles.css [hunk 4]
```diff
@@ -397,11 +397,16 @@ a.dropdown-item:hover {
 
   .plugin-info {
     color: gray;
-    font-size: 0.75rem;
+    font-size: 0.8rem;
   }
 
   .plus-icon {
-    font-size: 0.70rem;
+    font-size: 1rem;
```
Comment (hemantgarg923) [inline src/styles.css:404]: We could define sizing classes for all bi icons similar to FogLAMP Manage: [https://github.com/dianomic/fogman-ui/pull/538/files#r1333119093](url)

**Review by mshariq-nerd — APPROVED** [review]

**Review by hemantgarg923 — APPROVED** [review]

**Pull request merged**